src/pages/iou/IOUCurrencySelection.js

```diff
@@ -51,7 +57,12 @@ const propTypes = {
 };
 
 const defaultProps = {
-    myPersonalDetails: {preferredCurrencyCode: CONST.CURRENCY.USD, preferredCurrencySymbol: '$'},
+    myPersonalDetails: {
+        preferredCurrencyCode: CONST.CURRENCY.USD,
+        preferredCurrencySymbol: '$',
+        selectedCurrencyCode: CONST.CURRENCY.USD,
+        selectedCurrencySymbol: '$',
```

Comment:
What is the difference between selected and preferred? It seems like we would want to persist the former but not the latter.

Reply:
Preferred is the default fetched based on user location/IP/policy. Selected could be changed by the user in each IOU.

For example, I'm based in Canada so my `preferredCurrency` is CAD. However, if I wanted to send money in USD, I would initiate an IOU (defaults to CAD) and then set the `selectedCurrency` to USD.

So in that case, we want to persist `preferredCurrency` and not `selectedCurrency`.

Reply:
>if I wanted to send money in USD, I would initiate an IOU (defaults to CAD) and then set the selectedCurrency to USD

And in this case, once you set that value do we ever use it again? Or will you have to set it again next time?

>So in that case, we want to persist preferredCurrency and not selectedCurrency.

🤔

Reply:
I think the idea is that each new IOU starts with your `preferredCurrency`, because most transactions are likely to be in that currency. However, you can change the currency if it is different for that specific IOU.

So in that case, I would set it to USD for that one IOU, but after confirming/dismissing it the next IOU would start off with CAD. That is the current logic we are following.

Reply:
Ok do you agree we don't really need to store the `selectedCurrency` in `myPersonalDetails` then because it's not really a user preference and more of a temporary state that we no longer care about as soon as we are done creating the IOU?

Reply:
Maybe `preferredCurrency` should be `localCurrency` instead? Preferred sounds a bit like user preference, whereas this is simply the user's current local currency (based on IP or GPS).

Reply:
Makes sense to me, but if it's already what the API expects I wonder if it's worth changing...

Reply:
Maybe we should store the "selected currency" under the IOU key for Onyx? 
Unless we are planning to use this preference for all things moving forward once it's set.

Reply:
Done!